security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -341,6 +342,7 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 					conIDresourceNamePrefix, discReq.Node.Id, err)
 				return err
 			}
+			sdsServiceLog.Infoa("Pushed secret for ", discReq)
```

Comment:
Might want to make these in another PR if you want only my approval to merge this

Reply:
Right. Fixed.